extensions/binding/org.eclipse.smarthome.binding.tradfri.test/build.properties

```diff
@@ -0,0 +1,4 @@
+source.. = src/test/java/
+output.. = target/classes
```

Comment:
There was recently another discussion about `target/classes` vs `target/test-classes` I don't remember the outcome